views/js/qtiCreator/editor/ckEditor/htmlEditor.js

```diff
@@ -628,14 +627,13 @@ define([
          * @param {String} data
          */
         setData : function($editable, data) {
-            var editor = $editable.data('editor');
-            if(editor){
+            const editor = $editable.data('editor');
+            if (editor) {
                 if (_.isString(data)) {
                     editor.setData(_.escape(data));
                 }
-            }else{
-                throw new Error('no editor attached to the DOM element');
             }
+            return false
```

Comment:
**nitpick:** the function will always return `false`, no matter it performed the action or not.

Reply:
ok